packages/strategy-tests/src/lib.rs

```diff
@@ -566,11 +569,31 @@ impl Strategy {
                         document_type,
                         contract,
                     }) => {
+                        // Get the first 10 identities who are eligible to submit documents for this contract
+                        let first_10_eligible_identities: Vec<Identity> = current_identities
+                            .iter()
+                            .filter(|identity| {
+                                mempool_document_counter
+                                    .get(&(identity.id(), contract.id()))
+                                    .unwrap_or(&0)
```

Comment:
unwrap_or_default?

Reply:
Sure, but I will have to use .get().copied() because it's a &u64

I will do that.